.github/workflows/validate-rhel-8.yml

```diff
@@ -0,0 +1,76 @@
+# Run rhel-8 unit tests in a PR triggered by a "/test [...]" comment from an organization member.
+# This avoids running untrusted and unreviewed code on self-hosted runners.
+name: unit-tests-rhel-8
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  unit-tests:
+    # restrict running of tests to trusted organization members
+    # see https://developer.github.com/v4/enum/commentauthorassociation/
+    if: startsWith(github.event.comment.body, '/test') && contains('OWNER MEMBER', github.event.comment.author_association)
+    runs-on: [self-hosted, ci-tasks, rhel-8]
+    steps:
+      - name: Get information for pull request
+        uses: octokit/request-action@v2.x
+        id: pr_info
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      #- name: Dump PR info
```

Comment:
Before merge please remove these comments.

Reply:
Ack, done. I thought I'd leave it in, as it's really useful for debugging/development (and it took me > 1 minute to figure this out).